caf-logging-tomcat-juli/pom.xml

```diff
@@ -144,6 +144,7 @@
                                         <include>org/apache/juli/ClassLoaderLogManager**</include>
                                         <include>org/apache/juli/FileHandler**</include>
                                         <include>org/apache/juli/AsyncFile**</include>
+                                        <include>org/apache/juli/OneLineFormatter**</include>
```

Comment:
Why is this showing up in the PR when it was already in the merged PR: https://github.com/CAFapi/opensuse-tomcat-image/pull/42

I don't think we should be restoring already merged branches to make additional changes.

Reply:
It does not affect the history, we only have two commits, the original PR and the update to the release notes.